problems/removeOddNumbers.js

```diff
@@ -2,8 +2,13 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - The input array with all odd number removed
 */
-function removeOddNumbers() {
-
+function removeOddNumbers(arr) {
+let newArr = arr.filter(el => {
+    if(el % 2 === 0){
+        return arr[el]
+    }
+})
+return newArr
 }
```

Comment:
Can also return on line 6 because we don't use newArr for anything besides returning.